grippers:
  arm:
    type: GenericGripper
    args:
      name: gripper
      action_ns: gripper_controller/gripper_action
      base_link: wrist_roll_link
      tip_link: gripper_link
      timeout: 10.0
      min_gap: 0.0
      max_gap: 0.1
      max_effort: 0.0

cameras:
  head_camera: DepthCamera

moveit_pose_reference_frame: base_link
moveit_eef_step: 0.05
